src/nemo_evaluator/solvers/harbor.py

```
@@ -1564,33 +1626,74 @@ def _patch_terminus_cle_reset() -> None:
+    if "_EVALUATOR_LENGTH_ERROR_DETAILS" not in litellm_src:
+        anchor = "                truncated_response=content,\n            )\n            raise exc\n"
+        replacement = anchor.replace(
+            "            raise exc\n",
+            "            # _EVALUATOR_LENGTH_ERROR_DETAILS\n"
+            "            exc.llm_usage = usage_info\n"
+            '            exc.llm_model_name = response.get("model")\n'
+            '            exc.llm_reasoning_content = reasoning_content or message.get("reasoning")\n'
+            "            raise exc\n",
+        )
+        if litellm_src.count(anchor) != 1:
+            raise RuntimeError("Cannot patch Harbor LiteLLM.call length-error details; lite_llm.py has diverged.")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)